flipperdevices/flipperzero-firmware
# NFC: Fix gen1 writing with invalid BCC (lost fix from PR #2511)

# What's new

- Returned the lost fix from PR #2511 

# Verification 

From old PR:

- Try writing gen1a tag with invalid BCC (not readable via Flipper NFC app, but visible in 'Check Magic tag'). Tag should be writable.
- Try writing gen1a tag with valid BCC. Tag also should be writable.
- Try writing non gen1a tag. App should show that the card isn't magic.

New:

- Try writing gen4 tag

# Checklist (For Reviewer)
